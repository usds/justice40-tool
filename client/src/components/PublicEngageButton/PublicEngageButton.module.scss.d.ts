declare namespace PublicEngagementButton {
    export interface IPublicEventScss {
        tag: string;
        tagContainer: string;
        container: string;
        link: string;
    }
  }

declare const PublicEventScssModule: PublicEngagementButton.IPublicEventScss & {
    /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
    locals: PublicEngagementButton.IPublicEventScss;
  };

  export = PublicEventScssModule;
