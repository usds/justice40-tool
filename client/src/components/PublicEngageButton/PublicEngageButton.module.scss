@use '../../styles/design-system.scss' as *;

.tagContainer {
    align-self: flex-end;
    @include u-margin-bottom(1);
    @include u-margin-right(1);
    .tag {
        // @include u-bg("gray-cool-10"); // Standard gray
        @include u-bg("yellow-20v"); // "new" yellow
        color: black;
        border-radius: 5px;
        @include u-text('bold');
    }
}

.container {
    display: flex;
    flex-direction: column;
    @include u-padding-top(2.5);
    @include u-margin-top(4);

    .link, .link:visited {
        color:white;
        text-decoration: none;

        .engagementButton{
            // @include u-text("blue-70v");
            // @include u-bg("yellow-20v");
            height: 40px; 
            
            // &:hover {
            //   @include u-bg("yellow-20");
            //   @include u-text("gray-90");
            // }
      
            .buttonContainer{
              display: flex;
            
            .buttonImage{
                @include u-margin-right(1);
                width: 21px;
                margin-top: -3px;
      
                filter: invert(100%) sepia(100%) saturate(0%) hue-rotate(119deg) brightness(105%) contrast(101%);
              }
            }
          }
    }

}