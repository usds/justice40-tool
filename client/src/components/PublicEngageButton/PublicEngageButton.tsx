import React from 'react';
import {useIntl, Link} from 'gatsby-plugin-intl';
import {
  Button,
  // Tag
} from '@trussworks/react-uswds';

import * as styles from './PublicEngageButton.module.scss';
import * as PUBLIC_ENG_COPY from '../../data/copy/publicEngage';

// @ts-ignore
import eventIcon from '/node_modules/uswds/dist/img/usa-icons/event.svg';

const PublicEngageButton = () => {
  const intl = useIntl();

  return (
    <div className={styles.container}>
      {/* Remove Updated Tag for now */}
      {/* <div className={styles.tagContainer}>
        <Tag className={styles.tag}>
          {intl.formatMessage(PUBLIC_ENG_COPY.PUBLIC_ENG_BUTTON.TAG_LABEL)}
        </Tag>
      </div> */}
      <Link className={styles.link} to={'/public-engagement'}>
        <Button
          type="button"
          className={styles.engagementButton}
        >
          <div className={styles.buttonContainer}>
            <img
              className={styles.buttonImage}
              src={eventIcon}
              alt={intl.formatMessage(PUBLIC_ENG_COPY.PUBLIC_ENG_BUTTON.IMG_ICON_ALT_TAG)}
            />
            <div>
              {intl.formatMessage(PUBLIC_ENG_COPY.PUBLIC_ENG_BUTTON.LABEL)}
            </div>
          </div>
        </Button>
      </Link>
    </div>
  );
};

export default PublicEngageButton;
