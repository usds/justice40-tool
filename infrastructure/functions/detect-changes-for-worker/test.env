REGION=us-east-1
STAGE=sit

# ESC Cluster name that will from the containers
ECS_CLUSTER=j40-sit-justice40-data-harvester-ECSCluster-ktXGGU9zjwkb

# VPC Private Subnet that has a NAT GAteway
VPC_SUBNET_ID=subnet-07e68cb57322f7b1f

# Names of the container and task names.
GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe

# AWS Credentials
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1