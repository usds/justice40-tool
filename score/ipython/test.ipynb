{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1a4c0c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "70b3a793",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('data/fips_states.csv')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c514aad8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fips</th>\n",
       "      <th>state_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alabama</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alaska</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>4</td>\n",
       "      <td>Arizona</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>Arkansas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>6</td>\n",
       "      <td>California</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   fips  state_name\n",
       "0     1    Alabama \n",
       "1     2     Alaska \n",
       "2     4    Arizona \n",
       "3     5   Arkansas \n",
       "4     6  California"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ee44d9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
