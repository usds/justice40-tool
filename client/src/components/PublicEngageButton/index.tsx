import PublicEngageButton from './PublicEngageButton';

export default PublicEngageButton;
