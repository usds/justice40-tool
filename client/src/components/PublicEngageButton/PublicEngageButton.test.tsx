import * as React from 'react';
import {render} from '@testing-library/react';
import {LocalizedComponent} from '../../test/testHelpers';
import PublicEngageButton from './PublicEngageButton';

describe('rendering of the PublicEngageButton', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <PublicEngageButton/>
      </LocalizedComponent>,
  );

  it('checks if component renders', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
