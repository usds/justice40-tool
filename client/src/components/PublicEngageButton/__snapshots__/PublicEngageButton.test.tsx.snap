// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of the PublicEngageButton checks if component renders 1`] = `
<DocumentFragment>
  <div>
    <a
      href="/en/public-engagement"
    >
      <button
        class="usa-button"
        data-testid="button"
        type="button"
      >
        <div>
          <img
            alt="an icon that represents a calendar"
            src="test-file-stub"
          />
          <div>
            Public engagement
          </div>
        </div>
      </button>
    </a>
  </div>
</DocumentFragment>
`;
